mobile.css

```diff
@@ -1,4 +1,4 @@
-@media screen and (max-width: 360px) {
+@media only screen and (min-width: 320px) and (max-width: 480px) {
```

Comment:
- [ ] Good job for putting your project this far. However, there're few things you need to get right. It is required to use media queries for the Desktop breakpoint in the CSS file. (the value for min-width property should be in range of 768px or wider). Something like this `@media only screen and (min-width: 768px)`. This simply means that your desktop version starts from 768px and above and your mobile version is from 767px and below. This two versions must be responsive across every breakpoints within its peculiar ranges. At this moment, your project has not met this requirement. Kindly take your time and work it out. Best of luck.

### Your Design
[your-page.webm](https://user-images.githubusercontent.com/26597432/215133111-cb456352-4e64-4831-978c-d8b6bca2d84a.webm)